Project flow/Project setup checklist.md

```diff
@@ -18,9 +18,10 @@ GitHub repository creation is limited to DevOps only. Head to the [#devops-hotli
 
 To help them out, please use this form:
 
-- New repository name: **ios-clientname-projectname** (e.g. ios-hpb-mhpb)
+- New repository name: **[ios]-[clientname]-[projectname]** (e.g. ios-hpb-mhpb)
```

Comment:
```suggestion
- New repository name: **ios-[clientname]-[projectname]** (e.g. ios-hpb-mhpb)
```
It looks a bit better maybe without it as it's inside the iOS handbook so we can assume we will always use ios prefix 🤔